### PR TITLE
remove alias for ASEOpenMMRunner

### PR DESCRIPTION
**Pull request by Ragzouken**
closes https://github.com/IRL2/nanover-protocol/issues/64

### CODE REVIEW

**Comment by Ragzouken** [comment]
Not sure about the failing tests, they look like port conflicts but it only seems to be affecting this branch atm

File: python-libraries/nanover-ase/tests/openmm_ase/test_ase_runner.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def logging_params(tmp_path):
     return params
 
 
-@pytest.fixture(params=(ASEOpenMMRunner, OpenMMIMDRunner))
+@pytest.fixture(params=(ASEOpenMMRunner,))
```
Comment (jbarnoud) [inline python-libraries/nanover-ase/tests/openmm_ase/test_ase_runner.py:64]: We do not need the parametrized fixture anymore. We can get rid of `runner_class` and change `runner` to directly call `ASEOpenMMRunner`.

**Comment by jbarnoud** [comment]
Regarding the test, it looks like there is one test that messes up the others; every time we change the tests, we take the risk of changing the failing test. There may be a way to figure out what test does not clean up after itself... Maybe pytest can run a function after tear down to check if the port is in use?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Parallel tests
         run: python -m pytest --cov nanover python-libraries -n auto -m 'not serial'
       - name: Serial tests
-        run: python -m pytest --cov nanover python-libraries -n auto -m 'serial'
+        run: python -m pytest --cov nanover python-libraries -n 1 -m 'serial'
```
Comment (Ragzouken) [inline .github/workflows/tests.yml:29]: This is probably our issue: serial tests need to be run in a single process

Comment (jbarnoud) [reply .github/workflows/tests.yml:29]: Damn copy-paste! It is odd it did not blow up earlier!

**Comment by jbarnoud** [comment]
One last issue I see is that the openmm_graphene notebook shows errors. They are not due to the change from the PR, but this notebook contains many widgets that only are relevant in a live notebook. We should clear the output before merging.

**Pull request merged**